fiboa_cli/datasets/dk.py

```diff
@@ -0,0 +1,163 @@
+PROVIDER_URL = "https://geodata-info.dk/srv/eng/catalog.search#/metadata/d91b2c99-d9b0-4e6d-b323-20ac80548186"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "The Danish Agency for Agriculture"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC0-1.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {    
+    'MARKBLOKNR' : 'id', # fiboa core field , TODO MARKBLOKNR-id conversion is needed.     
+    'GEOMETRISK' : 'area',#fiboa custom field i.e total area = eligible area + ineligible area
+    'TARAAREAL': 'ineligible_area', #fiboa custom field
+    'STOETPROC' : 'support_proc',  # fiboa custom field - translated. Supported Scheme. Currently all values are NaN
+    'GB_FRADRAG' : 'deduction', # fiboa custom field - translated 
+    'MB_TYPE' : 'block_type', #fiboa custom field
+    'GB_AREAL': 'eligible_area', # fiboa core field i.e. eligible area
+    'geometry':'geometry'# fiboa core field
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-01-01T00:00:00Z"
+}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+# Functions to migrate data in columns to match the fiboa specification.
+# Example: You have a column area_m in square meters and want to convert
+# to hectares as required for the area field in fiboa.
+# Function signature:
+#   func(column: pd.Series) -> pd.Series
+COLUMN_MIGRATIONS = {
+    'MB_TYPE': lambda column: column.map({'OMD' : 'Rotational crops', 'PGR' : 'Permanent grass', 'ING' : 'None', 'PAF' : 'Permanent crops', 'MIX' : 'Mixed permanent grass and arable land', 'VKS' : 'Plants under greenhouse/ nurseries /potted plants', 'LDP' : 'Afforestation'}),
+}
+
+# Filter columns to only include the ones that are relevant for the collection,
+# e.g. only rows that contain the word "agriculture" but not "forest" in the column "land_cover_type".
+# Lamda function accepts a Pandas Series and returns a Series or a Tuple with a Series and True to inverse the mask.
+COLUMN_FILTERS = {
+    'MB_TYPE': lambda col: (col.isin(["Rotational crops", "Permanent crops", "Plants under greenhouse/ nurseries /potted plants"]), True)
+}
+
+# Custom function to migrate the GeoDataFrame if the other options are not sufficient
+# This should be the last resort!
+# Function signature:
+#   func(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame
+MIGRATION = None
+
+# Schemas for the fields that are not defined in fiboa
+# Keys must be the values from the COLUMNS dict, not the keys
+MISSING_SCHEMAS = {
+    "properties": {
```

Comment:
Should some of the float fields be have a minimum: 0 constraint?

Reply:
Yes, good suggestion. We'll add the required constraints to the properties.